tools/codegen/core/experiments_compiler.py

```diff
@@ -394,6 +419,62 @@ def GenerateExperimentsHdr(self, output_file, mode):
             print(file=H)
             print(f"#endif  // {include_guard}", file=H)
 
+    def _GenerateExperimentsSrcForPlatform(self, platform, mode, file_desc):
+        print("namespace {", file=file_desc)
+        have_defaults = set()
+        for _, exp in self._experiment_definitions.items():
+            print(
+                "const char* const description_%s = %s;"
+                % (exp.name, ToCStr(exp.description)),
```

Comment:
For another cleanup, maybe: this file uses both `"%s" % var` formatting and `f"{var}"` formatting styles. This PR introduces more of both. One or the other should suffice, and consistency would be good.

Reply:
Thanks for approving. I'll make the cleanup in another PR incase we need to rollback